### ISSUE TITLE
`npm run generate` generates wrong npm-shrinkwrap.json file

### ISSUE THREAD

**Issue by timtong1982** [OfficeDev/office-ui-fabric-react#6196]
### Please provide a reproduction of the bug in a codepen:
 

npm -v
5.6.0

node -v
v8.11.4

1. sync to upstream/master latest
2. npm i
3. npm run generate
4. npm run build -- fail

### Bug Report 
 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

**Comment by timtong1982** [comment]
re-pro on 
win 10
OSX
Debian 9

**Comment by timtong1982** [comment]
update generated shrinkwrap file and **npm run build** log

[npm-shrinkwrap.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/2349595/npm-shrinkwrap.zip)
[2018-09-04T17_36_03_404Z-debug.log](https://github.com/OfficeDev/office-ui-fabric-react/files/2349604/2018-09-04T17_36_03_404Z-debug.log)

**Comment by timtong1982** [comment]
[@uifabric/utilities 10:36:03] Pass: jest (37.589s)
[@uifabric/utilities 10:36:03] Pass: tslint (37.596s)
src/Context.test.tsx:8:3 - error TS2345: Argument of type '{ isTest: Requireable<boolean>; }' is not assignable to parameter of type 'ValidationMap<{ isTest: boolean; }>'.
  Types of property 'isTest' are incompatible.
    Type 'Requireable<boolean>' is not assignable to type 'Validator<boolean>'.
      Types of property '[nominalTypeHack]' are incompatible.
        Type 'boolean | null | undefined' is not assignable to type 'boolean | undefined'.
          Type 'null' is not assignable to type 'boolean | undefined'.

 8   {
     ~
 9     isTest: PropTypes.bool
   ~~~~~~~~~~~~~~~~~~~~~~~~~~
10   },
   ~~~


src/Context.test.tsx:17:3 - error TS2322: Type '{ isTest: Requireable<boolean>; }' is not assignable to type 'ValidationMap<{ isTest: boolean; }>'.
  Types of property 'isTest' are incompatible.
    Type 'Requireable<boolean>' is not assignable to type 'Validator<boolean>'.

17   public static contextTypes: PropTypes.ValidationMap<{ isTest: boolean }> = {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
18     isTest: PropTypes.bool
   ~~~~~~~~~~~~~~~~~~~~~~~~~~
19   };
   ~~~~



[@uifabric/utilities 10:36:03] Error: ts (37.598s): Error: Command failed: node F:\src\office-ui-fabric-react\common\temp\node_modules\typescript\lib\tsc.js -outDir lib-commonjs -t es5 -m commonjs --pretty --inlineSources --sourceRoot ..\src


============[ @uifabric/utilities ]=[ Error ]=[ 38.343s ]============


6 of 23: [@uifabric/utilities] failed to build!
7 of 23: [@uifabric/fabric-website-resources] blocked by [@uifabric/utilities]!
8 of 23: [@uifabric/fabric-website] blocked by [@uifabric/utilities]!
9 of 23: [ssr-tests] blocked by [@uifabric/utilities]!
10 of 23: [@uifabric/styling] blocked by [@uifabric/utilities]!
11 of 23: [@uifabric/file-type-icons] blocked by [@uifabric/utilities]!
12 of 23: [@uifabric/experiments] blocked by [@uifabric/utilities]!
13 of 23: [@uifabric/dashboard] blocked by [@uifabric/utilities]!
14 of 23: [@uifabric/icons] blocked by [@uifabric/utilities]!
15 of 23: [@uifabric/charting] blocked by [@uifabric/utilities]!
16 of 23: [office-ui-fabric-react] blocked by [@uifabric/utilities]!
17 of 23: [@uifabric/example-app-base] blocked by [@uifabric/utilities]!
18 of 23: [@uifabric/variants] blocked by [@uifabric/utilities]!
19 of 23: [server-rendered-app] blocked by [@uifabric/utilities]!
20 of 23: [test-bundle-button] blocked by [@uifabric/utilities]!
21 of 23: [todo-app] blocked by [@uifabric/utilities]!
22 of 23: [vr-tests] blocked by [@uifabric/utilities]!

SUCCESS (6)
================================
@uifabric/build
@uifabric/foundation
@uifabric/jest-serializer-merge-styles
@uifabric/merge-styles
@uifabric/webpack-utils
office-ui-fabric-react-tslint
================================

BLOCKED (16)
================================
@uifabric/charting
@uifabric/dashboard
@uifabric/example-app-base
@uifabric/experiments
@uifabric/fabric-website
@uifabric/fabric-website-resources
@uifabric/file-type-icons
@uifabric/icons
@uifabric/styling
@uifabric/variants
office-ui-fabric-react
server-rendered-app
ssr-tests
test-bundle-button
todo-app
vr-tests
================================

FAILURE (1)
================================
@uifabric/utilities
================================

[@uifabric/utilities] [error] 'Returned error code: 1'

rush rebuild - Errors! (1 minute 26.8 seconds)

**Comment by timtong1982** [comment]
fixed in PR #6216

**Comment by kenotron** [comment]
fix is merged
